index.js

```diff
@@ -1,18 +1,14 @@
 var os = require('os')
 var path = require('path')
 
-var platform = os.platform()
-if (platform !== 'linux' && platform !== 'darwin' && platform !== 'win32') {
-  console.error('Unsupported platform.')
-  process.exit(1)
+var binaries = {
```

Comment:
Please use nested arrays or `Object.assign(Object.create(null), {…})` to make sure this works as expected.

Reply:
I am not sure if i understand correctly your concerns.
Replacing object literal with `Object.assign` calls wont improve readability and make sure it work "as expected"
Refactoring it with arrays wont help either

Reply:
Using an object literal, you create an object with a non-empty prototype chain. I would like to prevent weird edge cases (as an unlikely example, `platform` might be `toString`), so let's use an object with an empty prototype chain; `Object.assign(Object.create(null), {…})` does that.

Reply:
Now it is fixed when nodejs is compiled for `hasOwnProperty OS`